src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -180,21 +181,6 @@ const ReaderRevenueLinksRemote: React.FC<{
 		setSupportHeader,
 	] = useState<React.FC<SupportHeaderProps> | null>(null);
 
-	const [hasBeenSeen, setNode] = useHasBeenSeen({
```

Comment:
INSERT and VIEW event tracking will now be handled within the header module, see https://github.com/guardian/support-dotcom-components/pull/459